src/rgw/driver/sfs/sfs_bucket.cc

```diff
@@ -20,15 +20,17 @@ using namespace std;
 namespace rgw::sal {
 
 int SFStore::set_buckets_enabled(
-    const DoutPrefixProvider* dpp, std::vector<rgw_bucket>& buckets_to_enable,
-    bool enabled
+    const DoutPrefixProvider* dpp,
+    std::vector<rgw_bucket>& /*buckets_to_enable*/, bool /*enabled*/
 ) {
   ldpp_dout(dpp, 10) << __func__ << ": TODO" << dendl;
   return -ENOTSUP;
 }
 
 int SFStore::get_bucket(
-    User* u, const RGWBucketInfo& i, std::unique_ptr<Bucket>* result
+    User* /*u*/
+    ,
+    const RGWBucketInfo& i, std::unique_ptr<Bucket>* result
```

Comment:
(Optional)
Wild, how did clang-format come up with that?

Reply:
no idea, my guess is that the `*` followed by the comment made `clang-format` decide to do it that way for whatever reason.